src/ChannelServer/World/Character.cs

```diff
@@ -322,6 +322,15 @@ public void LevelUp()
 		}
 
 		/// <summary>
+		/// Increases character's level by one.
+		/// </summary>
+		/// <param name="amount"></param>
+		public void LevelUp()
+		{
+			LevelUp(1);
```

Comment:
This should be prefixed with `this.`, as per our coding conventions, but I'll fix that.